libs/agno/agno/os/router.py

```diff
@@ -1005,87 +1063,3 @@ async def get_workflow(workflow_id: str):
             name=workflow.name,
             description=workflow.description,
         )
-
-    @router.post("/workflows/{workflow_id}/runs")
-    async def create_workflow_run(
```

Comment:
Noooo we still need this for other cases. Just remove the background/connection_id stuff